src/main/steps/common/common.content.ts

```diff
@@ -10,7 +10,7 @@ const en = {
   applyForAdoption: 'Apply for a Service',
   applyForDissolution: 'Apply for a Service',
   feedback:
-    'This is a new service – your <a class="govuk-link" aria-label="Feedback link, This will open a new tab. You’ll need to return to this tab and continue with your application within 60 mins so you don’t lose your progress." href="#" target="_blank">feedback</a> will help us to improve it.',
+    "<b> Help us improve this service</b><br>This is a new service. Help us for others to improve it by <a  class='govuk-link' href='https://www.smartsurvey.co.uk/s/SurveyExit/?service=Specials' target=_blank>giving your feedback</a>",
```

Comment:
This should definitely be the feedback survey rather than the exit survey.

Reply:
Fixed

Reply:
The pageurl still has a hardcoded value.  Needs to be the actual page the user is on (dynamic value).